src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.logo__image {
```

Comment:
The class name '.logo__image' is used here, but in the HTML file, the class 'logo__moyo' is used for the image. Ensure consistency in class names between HTML and CSS.